src/components.d.ts

```diff
@@ -4,46 +4,59 @@
  * This is an autogenerated file created by the Stencil compiler.
  * It contains typing information for all components that exist in this project.
  */
-import { HTMLStencilElement, JSXBase } from "@stencil/core/internal";
+
+
+import { HTMLStencilElement, JSXBase } from '@stencil/core/internal';
+
+
 export namespace Components {
-    interface ManifoldPlanTable {
-        "baseUrl"?: string;
-        "clientId"?: string;
-        "ctaText"?: string;
-        "env"?: "stage" | "local" | "prod";
-        "gatewayUrl"?: string;
-        "productId"?: string;
-    }
+  interface ManifoldPlanTable {
+    'baseUrl'?: string;
+    'clientId'?: string;
+    'ctaText'?: string;
+    'env'?: 'stage' | 'local' | 'prod';
+    'gatewayUrl'?: string;
+    'productId'?: string;
+  }
```

Comment:
Was this edited manually? Or is this just the new version?

Reply:
It was automatic, I had the dev server running.